### PR TITLE
Added instance type for DEFAULT instance and updated NH index to have…

### PR DESCRIPTION
**Pull request by mohanasm**
1. Added network instance type for DEFAULT instance

2. Index 1-64 are used for programming NH which is used for prefixes in default instance. Updated NH index start value for vrf1(nhID1) from 65, vrf2(nhID2) from 1065 and vrf3(nhID3) from 18565

### CODE REVIEW

File: feature/gribi/ate_tests/gribi_scaling/gribi_scaling_test.go [hunk 1]
```diff
@@ -62,9 +62,9 @@ const (
 	IPBlock1      = "198.18.0.1/18"   // IPBlock1 represents the ipv4 entries in VRF1
 	IPBlock2      = "198.18.64.1/18"  // IPBlock2 represents the ipv4 entries in VRF2
 	IPBlock3      = "198.18.128.1/18" // IPBlock3 represents the ipv4 entries in VRF3
-	nhID1         = 2                 // nhID1 is the starting nh Index for entries in VRF1
-	nhID2         = 1002              // nhID2 is the starting nh Index for entries in VRF2
-	nhID3         = 18502             // nhID3 is the starting nh Index for entries in VRF3
+	nhID1         = 65                // nhID1 is the starting nh Index for entries in VRF1
```
Comment (greg-dennis) [inline feature/gribi/ate_tests/gribi_scaling/gribi_scaling_test.go:65]: Unless there's a good reason, lowercase "index" in each of these comments

**Review by greg-dennis — CHANGES_REQUESTED** [review]

File: feature/gribi/ate_tests/gribi_scaling/gribi_scaling_test.go (hunk 1 above)
Comment (mohanasm) [reply feature/gribi/ate_tests/gribi_scaling/gribi_scaling_test.go]: Makes sense to have 'index'. Renamed 'Index' to 'index'.

**Comment by xw-g** [comment]
changes SGTM, but I'm not sure why internal presubmit is failing. @sachendras is following up.

**Comment by sachendras** [comment]
Google internal confirmation in [b/262666507](https://b.corp.google.com/262666507)

**Comment by sachendras** [comment]
Google internal confirmation in b/263306234

**Review by sachendras — APPROVED** [review]

**Review by sachendras — APPROVED** [review]

**Review by sachendras — APPROVED** [review]

**Review by greg-dennis — APPROVED** [review]

**Pull request merged**